libs/aws/langchain_aws/vectorstores/__init__.py

```diff
@@ -0,0 +1,49 @@
+"""**Vector store** stores embedded data and performs vector search.
+
+One of the most common ways to store and search over unstructured data is to
+embed it and store the resulting embedding vectors, and then query the store
+and retrieve the data that are 'most similar' to the embedded query.
+
+**Class hierarchy:**
+
+.. code-block::
+
+    VectorStore --> <name>  # Examples: Annoy, FAISS, Milvus
+
+    BaseRetriever --> VectorStoreRetriever --> <name>Retriever  # Example: VespaRetriever
+
+**Main helpers:**
+
+.. code-block::
+
+    Embeddings, Document
+"""  # noqa: E501
+
+import importlib
+from typing import TYPE_CHECKING, Any
+
+if TYPE_CHECKING:
+    from langchain_core.vectorstores import (
+        VectorStore,  # noqa: F401
+    )
+
+    from langchain_aws.vectorstores.documentdb import (
+        DocumentDBVectorSearch,  # noqa: F401
+    )
+__all__ = [
+    "DocumentDBVectorSearch",
+]
+
+_module_lookup = {
+    "DocumentDBVectorSearch": "langchain_community.vectorstores.documentdb",
+}
+
+
+def __getattr__(name: str) -> Any:
+    if name in _module_lookup:
+        module = importlib.import_module(_module_lookup[name])
+        return getattr(module, name)
+    raise AttributeError(f"module {__name__} has no attribute {name}")
+
+
```

Comment:
What's the need for this setup?

Reply:
@3coins I copied the setup of `langchain_community/vectorstores/__init__.py`, but AFAIK it's redundant.

Reply:
In that case, we can just use a regular import for the `DocumentDBVectorSearch`, and remove the rest of the code.

Reply:
Done